classes/bridges/across.js

```diff
@@ -25,10 +25,41 @@ class AcrossBridge extends BaseBridge {
       integratorId: "0xaaff", // 2-byte hex string
       chains: [optimism, arbitrum, polygon, base],
     });
+    this.isInitialized = false;
+    this.feeCosts = null;
   }
+
   async init() {
     return;
   }
+
+  async fetchFeeCosts(
+    account,
+    fromChainId,
+    toChainId,
+    inputToken,
+    targetToken,
+    inputAmount,
+  ) {
+    const route = {
+      originChainId: fromChainId,
+      destinationChainId: toChainId,
+      inputToken: inputToken,
+      outputToken: targetToken,
+    };
+    const quote = await this.sdk.getQuote({
+      route,
+      inputAmount: inputAmount,
+    });
```

Comment:
we only need to getQuote once. So maybe you can have GPT to do that. Otherwise, it would be a performance issue in the future